staging/src/k8s.io/apiserver/pkg/authorization/authorizerfactory/builtin.go

```diff
@@ -31,7 +33,31 @@ func (alwaysAllowAuthorizer) Authorize(a authorizer.Attributes) (authorized bool
 	return true, "", nil
 }
 
-func NewAlwaysAllowAuthorizer() authorizer.Authorizer {
+func (alwaysAllowAuthorizer) RulesFor(user user.Info, namespace string) ([]rule.ResourceInfo, []rule.NonResourceInfo, error) {
```

Comment:
Seems like this could just be:
```go
return []rule.ResourceInfo{
    &rule.DefaultResourceInfo{/* ... */},
}, []rule.NonResourceInfo{
    &rule.DefaultNonResourceInfo{/* ... */},
}, nil
```

without all the appends.